From df0c886d88dc6549f5774eddaa5039db6294f19b Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megi@xff.cz>
Date: Sat, 21 May 2022 11:22:48 +0200
Subject: [PATCH 349/515] media: i2c: imx258: Don't fail probe when we can't
 identify the chip

I2C fails on my prototype, and after switch of wcam to isp1, this
would mean I'd lose LCD output. So, let's just report and ignore
the chip identification failure for now.

Signed-off-by: Ondrej Jirman <megi@xff.cz>
---
 drivers/media/i2c/imx258.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/media/i2c/imx258.c b/drivers/media/i2c/imx258.c
index 2570b51e48fc..885f45d1bd84 100644
--- a/drivers/media/i2c/imx258.c
+++ b/drivers/media/i2c/imx258.c
@@ -1358,8 +1358,8 @@ static int imx258_probe(struct i2c_client *client)
 
 	/* Check module identity */
 	ret = imx258_identify_module(imx258);
-	if (ret)
-		goto error_identify;
+	//if (ret)
+		//goto error_identify;
 
 	/* Set default mode to max resolution */
 	imx258->cur_mode = &supported_modes[0];
-- 
2.35.3

